### PR TITLE
Added game resolution argument when rendering the snapshot texture Th…

### PR DESCRIPTION
**Pull request by hardylr**
…is fixes antialiasing problem when using a game resolution > 1

### CODE REVIEW

File: src/core/content-snapshot.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export class ContentSnapshot extends Phaser.Image {
      * @extend Phaser.Image
      */
     constructor(game, x, y, noStage) {
-        const texture = new Phaser.RenderTexture(game, game.width, game.height);
+        const texture = new Phaser.RenderTexture(game, game.width, game.height, "", game.renderer.resolution);
```
Comment (cristianbote) [inline src/core/content-snapshot.js]: Hey @hardylr, thanks for submitting this PR!

I took a quick look, over the docs, to have a reference for your issue, and it looks like the fourth argument is the `key`, the fifth is the `scaleMode` and the sixth the `resolution`. I think you need to declare the `game.renderer.resolution` to be the sixth argument.

```js
const texture = new Phaser.RenderTexture(game, game.width, game.height, "", Phaser.scaleModes.DEFAULT, game.renderer.resolution);
```

File: src/core/content-snapshot.js [hunk 2]
```diff
@@ -46,4 +46,4 @@ export class ContentSnapshot extends Phaser.Image {
         // Capture all input events
         this.inputEnabled = true;
     }
-}
+}
```
Comment (cristianbote) [inline src/core/content-snapshot.js]: Also, can you revert this line end?

**Review by cristianbote — CHANGES_REQUESTED** [review]

File: src/core/content-snapshot.js (hunk 1 above)
Comment (hardylr) [reply src/core/content-snapshot.js]: Hey @cristianbote , you are correct. I messed up the argument order. Will fix it.

**Review by cristianbote — APPROVED** [review]
👍

**Pull request merged**